app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+def create_report(data_file_name: str, report_file_name: str):
+    supply, buy = 0, 0
+
+    with open(data_file_name, "r") as data_file:
+        with open(report_file_name, "w") as report_file:
+            data = data_file.read().split("\n")
+            for line in data:
+                if line != "":
```

Comment:
i'm not sure about this, but i think if line == "" python automaticly skip

Reply:
Nop!